yarn-project/acir-simulator/src/client/private_execution.test.ts

```diff
@@ -400,7 +400,8 @@ describe('Private Execution test suite', () => {
       const secret = new Fr(1n);
       const preimage = await buildL1ToL2Message([new Fr(bridgedAmount), new Fr(recipient.x)], contractAddress, secret);
 
-      const messageKey = preimage.hash();
+      // stub message key
+      const messageKey = Fr.random();
```

Comment:
It seems strange that we spend time on a preimage if we simply draw a random value for the entryKey down here. But looking at the nr seems like it is allowing passing it instead of computing it so seems like a testing thing.

Reply:
definitely a testing this!